sdk/version.go

```diff
@@ -60,3 +61,12 @@ func GetDefaultAppId() string {
 	// This is a function in case the id needs to be configurable.
 	return coreosId
 }
+
+const (
+	CoreOSEpoch = 1372636800
+)
+
+// GetCoreOSAge returns the number of days since the CoreOS epoch.
+func GetCoreOSAge() int64 {
+	return int64(time.Since(time.Unix(CoreOSEpoch, 0)).Seconds()) / 86400
```

Comment:
Converting from an int64 to a float64 to an int64 seems silly. How about:

```go
    time.Since(time.Unix(CoreOSEpoch, 0)) / (86400 * time.Second)
```